src/app/modules/record-viewer/record-viewer-styles.scss

```diff
@@ -9,3 +9,7 @@
   background-color: black;
   color: white
 }
+
+.record-content {
+  margin: 20px;
```

Comment:
We should really try to be more consistent and use increments of 8px. Like 8, 16, 24, 32